homeworks/17/chat-system/src/main/java/org/example/Server.java

```diff
@@ -0,0 +1,56 @@
+package org.example;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.security.InvalidParameterException;
+import java.util.Objects;
+
+public class Server {
+
+    private ServerSocket serverSocket;
+
+    public Server(ServerSocket serverSocket) {
+        setServerSocket(serverSocket);
+    }
+
+    public static void main(String[] args) throws IOException {
+        int port = 42069;
+        ServerSocket serverSocket = new ServerSocket(port);
+        Server server = new Server(serverSocket);
+        System.out.println("Port is " + port + ". Machkai tigre!");
```

Comment:
I don't think this is appropriate end of message. There is no need to show what port are we on.